lib/browser/rollup-legacy-main.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import '../zone';
+import '../common/promise';
+import '../common/to-string';
+import './api-util';
+import './browser-legacy';
```

Comment:
does the import order between `./browser-legacy` and `./browser` matter?

if not then you could just import `rollup-main.ts` and add `./browser-legacy`. 

that way it's clear that legacy build just adds that one import path...

Reply:
@IgorMinar , yes, we need to keep the `order`, I split a `rollup-common` to contain the `common part`.